src/backend/data/user.js

```diff
@@ -0,0 +1,30 @@
+const Feed = require('./feed.js');
+
+class User {
+  constructor(name, email, id) {
+    this.name = name;
+    this.email = email;
+    this.id = id;
+  }
+
+  toJSON() {
+    return {
+      name: this.name,
+      email: this.email,
+      id: this.id,
+    };
+  }
```

Comment:
Why is this necessary?

Reply:
This will override how the user object gets stored as JSON (e.g., JSON.stringify() will use it) when express writes this object to the session.